scico/linop/xray/astra.py

```diff
@@ -441,8 +443,15 @@ def convert_to_scico_geometry(
     linear system to determine the affine relationship between them.
 
     Args:
-        vol_geom: ASTRA volume geometry object.
-        proj_geom: ASTRA projection geometry object.
+        input_shape: Shape of the input array.
+        det_count: Number of detector elements. See the
+           `astra documentation <https://www.astra-toolbox.com/docs/geom3d.html#projection-geometries>`__
+           for more information.
+        det_spacing: Spacing between detector elements. See the
+           `astra documentation <https://www.astra-toolbox.com/docs/geom3d.html#projection-geometries>`__
+           for more information.
+        angles: Array of projection angles in radians.
+        vectors: Array of geometry specification vectors.
```

Comment:
Slightly confusing that you can specify both of these, but (I think) only one ever gets used.

Reply:
I'll make an issue to note that this needs to be addressed.